pysamstats/test/test_io.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import, print_function, division
+import logging
+
+from pysam import Samfile
+from pysamstats import load_coverage, stat_coverage
+
+from pysamstats.io import write_hdf5
+import tables
+
+logger = logging.getLogger(__name__)
+debug = logger.debug
+
+
+def write_hdf5_dtype(arg):
+
+    # testing auto dtype determination.
+    dtype, alignment, result, label = arg
+    import tempfile
+
+    # use auto
+    tmp = tempfile.mktemp(suffix=".h5")
+    write_hdf5("coverage", tmp, alignment, chrom=label, dtype=dtype)
+    with tables.open_file(tmp, mode="r") as h5file:
+        assert result == h5file.root.data.dtype["chrom"].itemsize
+
+
+def test_write_hdf5():
```

Comment:
I'd be inclined to call this something like "test_write_hdf5_chrom_dtype" just to be descriptive.